api/v1alpha2/clientmount_types.go

```diff
@@ -189,6 +189,7 @@ type ClientMountStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:storageversion
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="ERROR",type="string",JSONPath=".status.error.severity"
```

Comment:
When you add custom printcolumns, then you also have to manually add one for AGE, otherwise it won't be displayed.
Usually AGE is the last column.